src/audio/VoiceRecording.ts

```diff
@@ -32,12 +33,26 @@ import mxRecorderWorkletPath from "./RecorderWorklet";
 
 const CHANNELS = 1; // stereo isn't important
 export const SAMPLE_RATE = 48000; // 48khz is what WebRTC uses. 12khz is where we lose quality.
-const BITRATE = 24000; // 24kbps is pretty high quality for our use case in opus.
 const TARGET_MAX_LENGTH = 900; // 15 minutes in seconds. Somewhat arbitrary, though longer == larger files.
 const TARGET_WARN_TIME_LEFT = 10; // 10 seconds, also somewhat arbitrary.
 
 export const RECORDING_PLAYBACK_SAMPLES = 44;
 
+interface RecorderOptions {
+    bitrate: number;
+    encoderApplication: number;
+}
+
+export const voiceRecorderOptions: RecorderOptions = {
+    bitrate: 24000,
+    encoderApplication: 2048,
+};
+
+export const higQualityRecorderOptions: RecorderOptions = {
```

Comment:
Minor, but I think we have a typo here, should it be `high`?

Reply:
Fixed.